src/Behat/Config/Formatter/JUnitFormatter.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config\Formatter;
+
+final class JUnitFormatter extends Formatter
+{
+    public const NAME = 'junit';
+
+    public function __construct(
+        bool $timer = true,
```

Comment:
You're right, the option does not work, but it seems like a bug.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="frontend_managing_account_organization">
  <testsuite name="Editing an organization" tests="1" skipped="0" failures="0" errors="0" time="0.999">
    <testcase name="Renaming an organization" classname="Editing an organization" status="passed" time="0.981" file="features/frontend/account/organization/editing.feature"></testcase>
  </testsuite>
  <testsuite name="Switching organizations" tests="1" skipped="0" failures="0" errors="0" time="1.079">
    <testcase name="Switching to a different organization" classname="Switching organizations" status="passed" time="1.067" file="features/frontend/account/organization/switching_organizations.feature"></testcase>
  </testsuite>
  <testsuite name="Editing an organization" tests="5" skipped="0" failures="0" errors="0" time="2.082">
    <testcase name="Postal code must be at least 5 characters" classname="Editing an organization" status="passed" time="0.405" file="features/frontend/account/organization/validation.feature"></testcase>
    <testcase name="Postal code must contains only numbers" classname="Editing an organization" status="passed" time="0.402" file="features/frontend/account/organization/validation.feature"></testcase>
    <testcase name="City must contains only text" classname="Editing an organization" status="passed" time="0.368" file="features/frontend/account/organization/validation.feature"></testcase>
    <testcase name="Phone number must be at least 10 characters" classname="Editing an organization" status="passed" time="0.369" file="features/frontend/account/organization/validation.feature"></testcase>
    <testcase name="Email must be valid" classname="Editing an organization" status="passed" time="0.480" file="features/frontend/account/organization/validation.feature"></testcase>
  </testsuite>
</testsuites>
```

Reply:
I've removed the option for now.

Reply:
The `time` attribute is a required part of the junit XML schema I believe, so probably correct that we are always filling it.

In the other formatters AFAIK the `timer` option controls whether to print statistics on time / step counts after the run - to be honest I'm not sure why you'd need or want to turn that off but the option exists so...

But for JUnit it wouldn't make sense to print anything that's not part of the XML so I think maybe this was just added by mistake / because maybe at one point all formatters took a `timer` or something.

I think we can definitely remove from the object. We maybe need to leave it in the JUnitFormatterFactory options for BC in case people are currently passing it - not sure if those parameters get validated in any way.